warning C4819 and error C2220

Hello,

I have already git clone the newest version of universal-ctags. But the warning C4819 and error C2220 happen for some files as below. It can be solved by modify the file on localy but I suggest it can be solved from here and everyone can have it.
![image](https://user-images.githubusercontent.com/20179047/115939197-f1644d00-a4cf-11eb-84a4-488637b11629.png)

--- a/parsers/verilog.c
+++ b/parsers/verilog.c
@@ -1672,7 +1672,7 @@ static int processType (tokenInfo* token, int c, verilogKind* kind, bool* with)
 }
 
 // class_type ::=
-//       ps_class_identifier [ # ( … ) ] { :: class_identifier [ # ( … ) ] }
+//       ps_class_identifier [ # ( ... ) ] { :: class_identifier [ # ( ... ) ] }
 // "interface_identifier ." is also handled
 static int skipClassType (tokenInfo* token, int c)
 {
@@ -1714,7 +1714,7 @@ static int skipClassType (tokenInfo* token, int c)
 //   ...
 //   | virtual [ interface ] identifier [ # ( [ ... ] ) ]  [ . identifier ]
 //   | [ class_type :: | identifier :: | $unit :: ] identifier { [ ... ] }
-//   | [ identifier :: | $unit :: ] identifier [ # ( … ) ] { :: identifier [ # ( ... ) ] }
+//   | [ identifier :: | $unit :: ] identifier [ # ( ... ) ] { :: identifier [ # ( ... ) ] }
 //   | ...
 //
 //   mayPortDecl: may be a ANSI port declaration.  true for module, interface, or program.

